### PR TITLE
CVE-2014-0474 and CVE-2017-12794 

### PR DESCRIPTION
**Pull request by mxb9771**

### CODE REVIEW

File: cves/CVE-2014-0474.yml [hunk 1]
```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+    While logic was being implemented for FilePathField, GenericIPAddressField,
```
Comment (clairevc99) [inline cves/CVE-2014-0474.yml]: I wouldn't include the specific field names, but elaborate more on the general issue, as you did in the second sentence. I would also proofread these lines, there are a couple grammatical mistakes.

File: cves/CVE-2014-0474.yml [hunk 2]
```diff
@@ -146,10 +149,13 @@ discovered:
...
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    There was no evidence to where this was found. I looked on the original report where
+    the bug was first mentions. It was a sole individual who I could not confirm or deny works
+    for Google
+  automated: nil
```
Comment (clairevc99) [inline cves/CVE-2014-0474.yml]: I had an issue with nil for some keys and suggest putting false instead

File: cves/CVE-2014-0474.yml [hunk 3]
```diff
@@ -146,10 +149,13 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    There was no evidence to where this was found. I looked on the original report where
+    the bug was first mentions. It was a sole individual who I could not confirm or deny works
```
Comment (clairevc99) [inline cves/CVE-2014-0474.yml]: Please proofread

File: cves/CVE-2014-0474.yml [hunk 4]
```diff
@@ -177,8 +185,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    There was an edge case that was not covered with the implementation
```
Comment (clairevc99) [inline cves/CVE-2014-0474.yml:192]: I would argue that testing isn't part of requirements, and that there was insufficient test cases, since the functionality already existed.

File: cves/CVE-2014-0474.yml (hunk 1 above)
Comment (clairevc99) [inline cves/CVE-2014-0474.yml]: upvotes: 4

File: cves/CVE-2017-12794.yml [hunk 5]
```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+    there was a XSS vulnerability present in django. This was a lower
```
Comment (clairevc99) [inline cves/CVE-2017-12794.yml:54]: I would provide more information about the context of the XSS vulnerability within the system. You mention a force escape issue and cookies, but it isn't very clear.

Comment (clairevc99) [inline cves/CVE-2017-12794.yml:54]: upvotes: 1

**Review by clairevc99 — COMMENTED** [review]
I added a few comments about providing more details.

File: cves/CVE-2014-0474.yml [hunk 6]
```diff
@@ -15,24 +15,24 @@ curated_instructions: |
...
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2014-04-30
 published_instructions: |
   Is there a published fix or patch date for this vulnerability?
   Please enter your date in YYYY-MM-DD format.
-published_date:
+published_date: 2014-04-20
```
Comment (Aust1n46) [inline cves/CVE-2014-0474.yml]: The patch published date was before the date the vulnerability was reported to the security team?  Might want to double check that.

File: cves/CVE-2014-0474.yml [hunk 7]
```diff
@@ -207,10 +216,15 @@ interesting_commits:
...
-    note:
+  - commit: aa80f498de6d687e613860933ac58433ab71ea4b
+    note: |
+        I find this commit interesting because this is one of the main solutions
+        to the problem. The problem fixed with this commit is a simple edge case.
+  - commit: ce3c281090320172d22e8a6057250d103c93438e
+    note: |
+        This is the commit the caused the problem mentioned in the above interesting
+        commit. I am also including this because it is important to remember to
+        do basic edge cases in your functions.
```
Comment (Aust1n46) [inline cves/CVE-2014-0474.yml:231]: Do you mean "test basic edge cases"?  The wording here is a little unclear.

File: cves/CVE-2017-12794.yml [hunk 8]
```diff
@@ -281,8 +303,9 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 79
+CWE_note: |
+    Due to an auto escape problem this gave the user to XSS
```
Comment (Aust1n46) [inline cves/CVE-2017-12794.yml:311]: Unclear wording here as well.  Allowed the user to XSS?

File: cves/CVE-2017-12794.yml [hunk 9]
```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```
Comment (Aust1n46) [inline cves/CVE-2017-12794.yml:53]: Upvotes: 2

File: cves/CVE-2014-0474.yml [hunk 10]
```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```
Comment (Aust1n46) [inline cves/CVE-2014-0474.yml:53]: Upvotes: 2

File: cves/CVE-2014-0474.yml (hunk 2 above)
Comment (Aust1n46) [reply cves/CVE-2014-0474.yml]: I also used false for those.

**Review by Aust1n46 — COMMENTED** [review]
Nice job!

File: cves/CVE-2014-0474.yml (hunk 2 above)
Comment (mxb9771) [reply cves/CVE-2014-0474.yml]: removed nil thanks for the suggestion

File: cves/CVE-2014-0474.yml (hunk 4 above)
Comment (mxb9771) [reply cves/CVE-2014-0474.yml:192]: I was unsure of this one but I believe that given a certain input it does not behave properly would be a specification or requirements issue

File: cves/CVE-2014-0474.yml (hunk 3 above)
Comment (mxb9771) [reply cves/CVE-2014-0474.yml]: fixed some errors. thank you

File: cves/CVE-2014-0474.yml (hunk 10 above)
Comment (mxb9771) [reply cves/CVE-2014-0474.yml:53]: thank you

File: cves/CVE-2014-0474.yml (hunk 6 above)
Comment (mxb9771) [reply cves/CVE-2014-0474.yml]: I think I switched dates thank you
